homeassistant/components/notify/pushover.py

```diff
@@ -63,10 +63,18 @@ def send_message(self, message='', **kwargs):
 
         targets = kwargs.get(ATTR_TARGET)
 
-        for target in targets:
-            if target is not None:
-                data['device'] = target
-
+        if targets is not None:
+            for target in targets:
+                if target is not None:
```

Comment:
The list will always contain a string according to [schema](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/notify/__init__.py#L45) so you can remove this check